test/use-package-test.el

```diff
@@ -1,4 +1,4 @@
-;;; use-package-tests.el --- Tests for use-package.el
+;;; use-package-test.el --- Tests for use-package.el
```

Comment:
Hmm, the Emacs convention is `<pkgname>-tests.el` not `-test.el`, does cask require this name change?